src/redis-cli.c

```diff
@@ -46,8 +46,8 @@
 #include <limits.h>
 #include <math.h>
 
-#include "hiredis.h"
-#include "sds.h"
+#include <hiredis.h>
+#include <sds.h> /* use sds.h from hiredis, so that only one set of sds functions will be present in the binary */
```

Comment:
we *can* also update the sds.c and sds.h inside hiredis. but i think we should keep the above change.
since hiredis has its own sds implementation, redis-cli should use it, rather than use the one that belongs to the redis-server.

otherwise, if/when there's a difference between the two implementations, redis-cli uses linline functions from one implementation and non-inline functions from the other.

please note that i also made a change in the Makefile to reflect that... although the sds.o there was actually unused (seems like the linker simply ignored it)